app/views/reservations/_form.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_with(model: reservation) do |form| %>
+  <% if reservation.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(reservation.errors.count, "error") %> prohibited this reservation from being saved:</h2>
+
+      <ul>
+        <% reservation.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :seance_id, style: "display: block" %>
+    <%= form.text_field :seance_id %>
+  </div>
+
+  <div>
+    <%= form.label :user_id, style: "display: block" %>
```

Comment:
Is it intended to allow the user to make a reservation in the name of another user? :thinking: 
Looking at the implementation of `ReservationsController#create` it looks like I can create a reservation for any user in the system.

Reply:
Sorry, I should have completely delete this method before making PR, it was just for testing and it's gonna look completely different when I'll implement tickets